specs/data_structures.md

```diff
@@ -166,6 +166,13 @@ Address is a [type alias](#type-aliases).
 
 Addresses are the [hash](#hashing) [digest](#hashdigest) of the [public key](#publickey).
 
+### PublicKey
```

Comment:
Should fix the ToC order too

Reply:
Actually, `PublicKey` seems to only have been used for evidence. Is there a need for it? Why remove `PublicKey` from equivocations? Because it can be recovered from the signature?

Reply:
The pubkey was used to get the address of the validator that committed the infraction but the votes contained the same info, so it was redundant. 

I can remove it entirely

Reply:
actually I think it may be better to key the pubkey becuase it is referenced elsewhere

Reply:
Ah right, votes have both an address and signature (so technically the address can be removed from there too, but would involve non-trivial changes to Tendermint, so we can revisit it later). The pubkey isn't used anywhere else, so probably best to just remove it.